ldelf/sub.mk

```diff
@@ -13,3 +13,4 @@ srcs-y += ta_elf.c
 srcs-y += ta_elf_rel.c
 srcs-$(CFG_FTRACE_SUPPORT) += ftrace.c
 srcs-$(CFG_TA_PAUTH) += pauth.c
+cflags-lib-$(CFG_TA_SANITIZE_UNDEFINED) += -fsanitize=undefined
```

Comment:
Is this needed? You already assign `cflags$(sm)` in `ldelf.mk`

Reply:
Removed. Done